src/ISFirmwareUpdater.cpp

```diff
@@ -129,9 +129,9 @@ bool ISFirmwareUpdater::fwUpdate_handleVersionResponse(const fwUpdate::payload_t
     target_devInfo = &remoteDevInfo;
 
     if(pfnInfoProgress_cb != nullptr) {
-        if ((remoteDevInfo.hardware >= HDW_TYPE__UINS) && (remoteDevInfo.hardware <= HDW_TYPE__GPX)) {
+        if ((remoteDevInfo.hardwareType >= DEV_INFO_HARDWARE_TYPE_UINS) && (remoteDevInfo.hardwareType <= DEV_INFO_HARDWARE_TYPE_GPX)) {
             const char *hdw_names[5] = {"UNKNOWN", "UINS", "EVB", "IMX", "GPX"};
```

Comment:
You didn't modify this line... but while we are in here, doing this hardwareType, we should move this line into global const in data_sets.h that defines the string constants for each hardware type (IMX, GPX, etc).  We have these string duplicated lots of places, and could simplify and be consistent elsewhere.  ie, in ISDisplay.cpp above, for example, by replacing the switch with a lookup of "g_constHdwTypeNames[info.hardwareType]".